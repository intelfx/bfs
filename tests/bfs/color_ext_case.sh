# *.gz=01;31:*.GZ=01;32         -- case sensitive
# *.tAr=01;33:*.TaR=01;33       -- case-insensitive
# *.TAR.gz=01;34:*.tar.GZ=01;35 -- case-sensitive
# *.txt=35:*TXT=36              -- case-insensitive
export LS_COLORS="*.gz=01;31:*.GZ=01;32:*.tAr=01;33:*.TaR=01;33:*.TAR.gz=01;34:*.tar.GZ=01;35:*.txt=35:*TXT=36"
bfs_diff rainbow -color
