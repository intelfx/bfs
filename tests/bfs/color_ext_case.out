[01;34m$'rainbow/\e[1m'[0m
[01;34m$'rainbow/\e[1m/'[0m$'\e[0m'
[01;34mrainbow[0m
[01;34mrainbow/[0m[01;31mlower.gz[0m
[01;34mrainbow/[0m[01;31mlower.tar.gz[0m
[01;34mrainbow/[0m[01;32mexec.sh[0m
[01;34mrainbow/[0m[01;32mupper.GZ[0m
[01;34mrainbow/[0m[01;32mupper.TAR.GZ[0m
[01;34mrainbow/[0m[01;33mlower.tar[0m
[01;34mrainbow/[0m[01;33mupper.TAR[0m
[01;34mrainbow/[0m[01;34mul.TAR.gz[0m
[01;34mrainbow/[0m[01;35mlu.tar.GZ[0m
[01;34mrainbow/[0m[01;35msocket[0m
[01;34mrainbow/[0m[01;36mbroken[0m
[01;34mrainbow/[0m[01;36mchardev_link[0m
[01;34mrainbow/[0m[01;36mlink.txt[0m
[01;34mrainbow/[0m[30;42msticky_ow[0m
[01;34mrainbow/[0m[30;43msgid[0m
[01;34mrainbow/[0m[33mpipe[0m
[01;34mrainbow/[0m[34;42mow[0m
[01;34mrainbow/[0m[36mfile.txt[0m
[01;34mrainbow/[0m[37;41msugid[0m
[01;34mrainbow/[0m[37;41msuid[0m
[01;34mrainbow/[0m[37;44msticky[0m
[01;34mrainbow/[0mfile.dat
[01;34mrainbow/[0mmh1
[01;34mrainbow/[0mmh2
